c7n/cli.py

```diff
@@ -52,6 +52,10 @@ def _default_options(p, blacklist=""):
             "-r", "--region", action='append', default=[],
             dest='regions', metavar='REGION',
             help="AWS Region to target.  Can be used multiple times")
+        provider.add_argument(
+            "--exclude-region", action='append', default=[],
+            dest="exclude_regions", metavar='EXCLUDE REGION',
+            help="AWS Region(s) to exclude.")
```

Comment:
we should say cloud region here, region is interpreted differently by provider, we shouldn't be introducing new aws specific language / terminology on the cli. with policy conditions its already possible to do this on the policy itself.